src/main.cpp

```diff
@@ -111,7 +112,14 @@ static bool _get_options(int argc, char **argv, Options &options)
 #endif
 #if defined DOCKER_COMPOSE_APP
 		("apps,a", po::value<string>(&options.apps),
-		 "Configure package-manager for this comma separate list of apps.")
+		"Configure package-manager for this comma separate list of apps.")
+		// --restorable-apps : turn ON Restorable Apps usage, its list == compose_apps or all Target apps
+		// --restorable-apps "app-01[,app-02]" : turn ON Restorable Apps usage, its list == UNION(compose_apps, app-01[,app-02])
+		// if --restorable-apps is not specified or `--restorable-apps ""` then it means turning restorable Apps off
+		 ("restorable-apps,A", po::value<string>(&options.restorable_apps)->implicit_value(" "),
+		 "Configure package-manager for this comma separate list of Restorable apps."
+		 "It's set to an empty list automatically if not specified and a system image is preloaded with Restorable Apps."
+		 "Restorable App list = UNION(compose-apps, restorable-apps),")
```

Comment:
I don't want to nit, but I've read this sentence a few times and the "if not specified" part keeps making me think you can't turn off restorable apps. i.e it defaults to all. Then I read your code comments and it becomes clear and expected. I think you should incorporate the comments into the usage message. They are more clear.

Reply:
Makes sense

Reply:
updated